packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -135,7 +161,9 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
 
     public constructor(
         protected readonly namespace?: string,
-        protected readonly properties?: ITelemetryLoggerPropertyBags) {
+        protected readonly properties?: ITelemetryLoggerPropertyBags,
+        public readonly supportsTags?: true | undefined) {
```

Comment:
No need to have ? and undefined on same line

Reply:
Also maybe parentLoggerSupportsTags or something more clearly articulating what it is about?

Reply:
This field should just be `abstract` on `TelemetryLogger` anyway